uw-frame-components/portal/messages/partials/announcement-mascot.html

```diff
@@ -0,0 +1,52 @@
+<!-- DESKTOP MASCOT ANNOUNCEMENTS -->
+<md-menu class="mascot-menu announcement-creeper"
+         ng-if="mode === 'mascot' && vm.announcements && vm.announcements.length > 0"
+         ng-class="{ 'announcement-hover' : vm.hover, 'announcement-active' : vm.active }"
+         md-offset="-200 0"
+         md-position-mode="target bottom">
+  <md-button class="mascot-button"
+             ng-click="vm.toggleActive();vm.toggleHover();$mdOpenMenu($event);"
+             ng-mouseenter="vm.toggleHover()"
+             ng-mouseleave="vm.toggleHover()"
+             aria-label="Click to see what's new"
+             md-no-ink
+             md-menu-origin>
+    <img ng-src="{{ vm.mascotImage }}"
+         ng-click="pushGAEvent('feature', 'toggle-bucky', 'na');">
+    <md-tooltip ng-if="!vm.active" md-direction="bottom" class="top-bar-tooltip" md-delay="500">
+      Click to see what's new ({{ vm.announcements.length }})
+    </md-tooltip>
+  </md-button>
+  <md-menu-content class="top-bar-menu-content">
+    <md-menu-item layout="row" layout-align="space-between center">
+      <span>Announcements</span>
+      <a href="features" ng-click="vm.markAllAnnouncementsSeen()" aria-label="see all announcements" class="link__see-all">
+        See all ({{ vm.announcements.length }})
+      </a>
+    </md-menu-item>
+    <md-menu-item ng-repeat="announcement in vm.announcements | orderBy:'-id' | limitTo:3"
```

Comment:
Will consider. Initial attempts to include it have resulted in bizarre errors (ex. dupes error when tracking by a field that definitely has no dupes). Like Dave said, it's not really a performance concern at the moment, but I'll keep trying to get it to work as long as this PR is still open :P